pkg/workflows/workflows_test.go

```diff
@@ -3,6 +3,7 @@ package workflows
 import (
 	"errors"
 	"fmt"
+	"github.com/Azure/draft/pkg/fixtures"
```

Comment:
nit: import order

Reply:
not specifically calling out this comment but I see lots of comments on PRs about import order in our repos. If that's something we actually care about and want to enforce we should have an automated linter enforcing it on our repos.

Just calling them out in PR comments is a longer feedback loop for something that's questionable to care about or enforce. I don't care about import order or import style personally but if we do it needs to be automated and if we don't want to do that it's likely not worth our time.

Reply:
Yeah to be honest I'm not sure why it's not enforced in our repos to begin with. I have my golang configured to auto-sort imports already since I've gotten a ton of feedback about it but agree that it's something that's easy enough to automate vs occupying developer time, both on the feedback and the implementation.

Reply:
getting a ton of feedback on it doesn't necessarily mean we have to enforce something. It's like an endless cycle